modules/programs/keepassxc.nix

```diff
@@ -45,12 +45,26 @@ in
     };
   };
 
-  config = lib.mkIf cfg.enable {
-    home.packages = lib.mkIf (cfg.package != null) [ cfg.package ];
-    xdg.configFile = {
-      "keepassxc/keepassxc.ini" = lib.mkIf (cfg.settings != { }) {
-        source = iniFormat.generate "keepassxc-settings" cfg.settings;
-      };
-    };
-  };
+  config = lib.mkIf cfg.enable (
+    lib.mkMerge [
+
+      {
+        xdg.configFile = {
+          "keepassxc/keepassxc.ini" = lib.mkIf (cfg.settings != { }) {
+            source = iniFormat.generate "keepassxc-settings" cfg.settings;
+          };
+        };
+      }
+
+      (lib.mkIf (cfg.package != null) {
+        home.packages = [ cfg.package ];
+        programs.brave.nativeMessagingHosts = [ cfg.package ];
+        programs.chromium.nativeMessagingHosts = [ cfg.package ];
+        programs.firefox.nativeMessagingHosts = [ cfg.package ];
+        programs.floorp.nativeMessagingHosts = [ cfg.package ];
+        programs.vivaldi.nativeMessagingHosts = [ cfg.package ];
```

Comment:
Just wondering if we want it to default to installing it as a native messaging host or if it should be gated behind a `enableBrowserIntegration` option.

Reply:
AFAIK, there is not downside to enabling the integration.

Reply:
Mostly just worry about people who don't like unexpected installations/configurations or can't disable stuff. But, I agree it is probably more confusing for the application to just not work with your browser...